internal/provider/waypoint/data_source_waypoint_application.go

```diff
@@ -100,6 +101,31 @@ func (d *DataSourceApplication) Schema(ctx context.Context, req datasource.Schem
 					},
 				},
 			},
+			"output_values": schema.ListNestedAttribute{
```

Comment:
Also aren't these typically set using the [output](https://developer.hashicorp.com/terraform/language/values/outputs) resource themselves? What does `output_values` give us that this existing resource doesn't have?

Reply:
Good question, I agree that this doesn't add functionality that doesn't exist there. The advantage of this is it displays the information about output values that is already present in the API responses anyways and is returned by Waypoint in the UI so that users can always retrieve the information from Waypoint the same way. I can definitely see the argument for how it is redundant in this case.